### DISCUSSION TITLE
Silent button available

### DISCUSSION THREAD

**Discussion by Corsair-63** [peterantypas/maiana#28] [Ideas]
Some transceivers for the yachting world uses to have the silent button,  should be available one?

I mean there would be some areas, mostly for dangerous areas either when you're fishing and don't want to share your position or another matter, that you don't want to transmit but you are receiving and for that option you have a switch to stop transmitting until you activate again.

**Comment by peterantypas** [comment]
Yes, the "TX OFF" signal is available. Starting with board revision 10.x, driving that signal to 3.3v disables transmission. The control box will have this feature.
